src/docs/product/releases/usage/sorting-filtering.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Sorting & Filtering
+sidebar_order: 20
+description: "Learn how to sort and filter releases."
+---
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+## Sorting Releases
+
+On the **Releases** page, you can use the "Sort By" dropdown to sort releases by:
+- Date Created
+- Total Sessions
+- [Active Sessions/Users](/product/releases/health/#active-sessionsusers)
+- [Crash Free Sessions/Users](/product/releases/health/#crash-free-sessionsusers)
+- Build Number
+- [Semantic Version](/platform-redirect/?next=/configuration/releases/%23bind-the-version)
+
+## Filtering Releases
+
+Search on the **Releases** page supports both raw text and query syntax. If you are using our [semantic versioning](/platform-redirect/?next=/configuration/releases/%23bind-the-version) format, the following query tokens are known to Sentry:
```

Comment:
```suggestion
Search on the **Releases** page, which supports both raw text and query syntax. If you are using our <PlatformLink to="/configuration/releases/#bind-the-version">Semantic Versioning</PlatformLink> format, you can use the following query tokens:
```

Reply:
Please also ignore this suggestion. I believe, however, that the anchor here is malformed. It should use `#bind-the-version`

Reply:
`%23` is urlencoded `#`. Without encoding it, the linkchecker github action would fail and the `#` would not survive the platform redirect.

Reply:
Ah, TIL. Thanks, Matej.